e2e/libs/setting/setting.py

```diff
@@ -1,73 +1,57 @@
 import os
+import time
+from utility.utility import logging
 from utility.utility import get_longhorn_client
-from setting.constant import SETTING_BACKUP_TARGET
-from setting.constant import SETTING_BACKUP_TARGET_CREDENTIAL_SECRET
-from setting.constant import SETTING_BACKUPSTORE_POLL_INTERVAL
+from utility.utility import get_retry_count_and_interval
 
 class Setting:
 
+    SETTING_BACKUP_TARGET = "backup-target"
+    SETTING_BACKUP_TARGET_CREDENTIAL_SECRET = "backup-target-credential-secret"
+    SETTING_BACKUPSTORE_POLL_INTERVAL = "backupstore-poll-interval"
```

Comment:
It seems that you're deleting the setting/constant.py file. Is there a conversation happening about this change?

Reply:
Look like `setting` didn't change in this PR. An additional ticket can be created if this modification is required.